front/app/containers/Admin/projects/project/files/components/UploadFileButtonWithModal.tsx

```diff
@@ -19,16 +19,16 @@ const UploadFileButtonWithModal = () => {
         buttonStyle="admin-dark"
         icon="plus-circle"
         iconSize="24px"
-        text={formatMessage(messages.addFile)}
+        text={formatMessage(messages.addFiles)}
         onClick={() => setModalOpen(true)}
       />
       <Modal
         opened={modalOpen}
         close={() => setModalOpen(false)}
-        width={'510px'}
+        width={'540px'}
```

Comment:
Why do we need hard-coded dimensions for this. We don't optimize the admin for mobile but still.. good to avoid, if possible.

Reply:
I was matching the width we had in the designs, but agreed that it doesn't feel necessary. A slightly wider modal should e fine :) I'll remove the width and double-check with Rob :+1: